src/main/scala/org/scalastyle/file/FixmeCommentChecker.scala

```diff
@@ -0,0 +1,47 @@
+package org.scalastyle.file
+
+import org.scalastyle.file.FixmeCommentChecker._
+import org.scalastyle.{ColumnError, FileChecker, Lines, ScalastyleError}
+
+/**
+ * Checks for line comment style FIXMEs
+ */
+class FixmeCommentChecker extends FileChecker {
```

Comment:
This seems to be exactly the same code as the FixmeCommentChecker - you could use the same abstract class for them both. Also, I'm not sure that we need both classes anyway - see general comments.